vmServer/test_macro/code/action.ps1

```diff
@@ -0,0 +1,4 @@
+$scriptpath = $MyInvocation.MyCommand.Path
```

Comment:
Suggest adding a top level directory named test and a sub directories named execute_macro and screenshot under them

Reply:
Change has been made